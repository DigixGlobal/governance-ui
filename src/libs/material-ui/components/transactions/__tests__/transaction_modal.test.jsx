import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import TransactionModal from '../transaction_modal';

describe('<TransactionModal />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <TransactionModal
        handleTransaction={jest.fn}
        trigger={<div />}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
