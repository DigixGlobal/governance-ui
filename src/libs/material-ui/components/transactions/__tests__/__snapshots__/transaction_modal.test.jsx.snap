// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransactionModal /> renders correctly 1`] = `
<span
  onClick={[Function]}
  role="button"
  tabIndex={0}
>
  <div />
</span>
`;
